app/services/analytics_events.rb

```diff
@@ -680,6 +680,7 @@ def idv_password_confirm_visited
   # User submitted IDV password confirm page
   def idv_password_confirm_submitted
     track_event('IdV: password confirm submitted')
+    idv_final(success: true)
```

Comment:
AFAIK this would be the only "compound" analytics logger, where previously, there's an expectation of 1:1 of method to tracked event.

Not sure if we are concerned about that, though I do sorta wonder if we should just update the affected dashboards instead, and consolidate to the one event?

Reply:
I had a hunch so I gave this a shot in 634d5455a7bb074e70152e8113da19c66d5536a7 ... if you don't like please feel free to revert/rebase it away!

Reply:
Actually, this is replicating the [previous behavior](https://github.com/18F/identity-idp/blob/main/app/controllers/idv/review_controller.rb#L49-L50) 😦 

I'll look at the lift to update dashboards.